src/easystar.js

```diff
@@ -16,6 +16,8 @@ const OPEN_LIST = 1;
 
 module.exports = EasyStar;
 
+EasyStar._nextId = 0;
```

Comment:
This variable doesn't need to be a property of EasyStar. You can simply change this line to `let nextInstanceId = 0;`